### PR TITLE
Add FAQ entry for control-plane

### PR DESCRIPTION
**Pull request by BrutalBirdie**
This fixes https://github.com/kubereboot/kured/issues/844

### CODE REVIEW

**Review by ckotzbauer — COMMENTED** [review]
Thanks for the addition!
The section is now specific to the "CriticalAddonsOnly" taint. It would be good if you can provide a guide on how to check which tains are available and then what needs to be done. (More like in your issue description). Otherwise people are wondering why their kured installation might not run on their control-plane if other taints are applied.

**Comment by BrutalBirdie** [comment]
I added a bit more on how to get taints.
Currently I got no good example with multiple taints and how to do that but it should be just more entries in the `tolerations`  or did I get that wrong 🤔

File: content/en/docs/faq.md [hunk 1]
```diff
@@ -42,7 +76,9 @@ tolerations:
         effect: NoSchedule
 ```
 
-In Order to make kured on the tainted nodes change this section into:
+In the docs about [taint and toleration](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) you an read up about the conecpt.
```
Comment (ckotzbauer) [inline content/en/docs/faq.md:79]: One small typo in "concept"

**Review by ckotzbauer — COMMENTED** [review]
Thanks, this looks pretty good!

**Comment by ckotzbauer** [comment]
Can you also please signoff all your commits?

**Review by ckotzbauer — APPROVED** [review]

**Comment by ckotzbauer** [comment]
Thanks a lot.

**Pull request merged**